x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/queries/TextExpansionQueryBuilder.java

```diff
@@ -51,6 +50,30 @@ public class TextExpansionQueryBuilder extends AbstractQueryBuilder<TextExpansio
     private SetOnce<TextExpansionResults> weightedTokensSupplier;
     private final TokenPruningConfig tokenPruningConfig;
 
+    public enum AllowedFieldTypesForTextExpansion {
```

Comment:
Nitpick: Since this enum is in the `TextExpansionQueryBuilder` we can shorten the name to something like `AllowedFieldType`. Also as a matter of convention using singular names for enums is typically preferential.

Reply:
Updated.